Cargo.toml

```diff
@@ -13,7 +13,7 @@ categories = ["development-tools::build-utils", "command-line-utilities"]
 
 [dependencies]
 anyhow = "1"
-tar = { version = "0.4.32", default-features = false }
-cargo_toml = "0.9.0"
-toml = "0.5.6"
-flate2 = { version = "1.0.3", default-features = false, features = ["zlib"] }
+tar = { version = "0.4", default-features = false }
+cargo_toml = "0.15"
+toml = "0.7"
+flate2 = { version = "1", default-features = false, features = ["zlib"] }
```

Comment:
Why did you remove the patch portion of the version requirement? Did you check that the crate works with the minimum versions that match your new requirement?

Reply:
Those were accidental, fixed :)